jooq/src/main/kotlin/com/terraformation/backend/jooq/Extensions.kt

```diff
@@ -4,10 +4,11 @@ import org.jooq.meta.jaxb.EmbeddableDefinitionType
 import org.jooq.meta.jaxb.EmbeddableField
 
 /** Converts "foo_bar_baz" to "fooBarBaz". */
-fun String.toCamelCase() = replace(Regex("_(.)")) { it.groupValues[1].capitalize() }
+fun String.toCamelCase() =
+    replace(Regex("_(.)")) { match -> match.groupValues[1].replaceFirstChar { it.uppercaseChar() } }
 
 /** Converts "foo_bar_baz" to "FooBarBaz". */
-fun String.toPascalCase() = toCamelCase().capitalize()
+fun String.toPascalCase() = toCamelCase().replaceFirstChar { it.uppercaseChar() }
```

Comment:
just so i understand, `capitalize` is deprecated but `replaceFirstChar` isn't?

Reply:
Yep. Not sure what the thinking was behind the deprecation, but that's right.